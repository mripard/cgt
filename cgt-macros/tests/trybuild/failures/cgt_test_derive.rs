#[derive(cgt_macros::cgt_test)]
struct Dummy;