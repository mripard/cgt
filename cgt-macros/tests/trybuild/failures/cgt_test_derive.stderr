error: expected derive macro, found attribute macro `cgt_macros::cgt_test`
 --> tests/trybuild/failures/cgt_test_derive.rs:1:10
  |
1 | #[derive(cgt_macros::cgt_test)]
  |          ^^^^^^^^^^^^^^^^^^^^ not a derive macro
  |
help: remove from the surrounding `derive()`
 --> tests/trybuild/failures/cgt_test_derive.rs:1:10
  |
1 | #[derive(cgt_macros::cgt_test)]
  |          ^^^^^^^^^^^^^^^^^^^^
  = help: add as non-Derive macro
          `#[cgt_macros::cgt_test]`

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/trybuild/failures/cgt_test_derive.rs:2:14
  |
2 | struct Dummy;
  |              ^ consider adding a `main` function to `$DIR/tests/trybuild/failures/cgt_test_derive.rs`
